stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -50,6 +50,11 @@ public class AttachmentGallery : ConstraintLayout {
 
     private var isFullScreen = false
 
+    /**
+     * Informs about index of the currently visible attachment
+     */
+    public val currentItemIndex: Int by lazy { binding.attachmentGallery.currentItem }
```

Comment:
Why is it lazy property instead of being just wrapped function like:
``` kotlin
public val currentItemIndex: Int
  get() = binding.attachmentGallery.currentItem
```
?

Reply:
Yep, getter is better. Fixed.